### PR TITLE
chore(docs): FilePermissions cas config user documentation

### PR DESCRIPTION
**Pull request by niladrih**
Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

**Why is this PR required? What issue does it fix?**:
This PR contains the user documentation for the FilePermissions feature in #125 

**What this PR does?**:
Adds docs/tutorials/file-permissions.md

**Does this PR require any upgrade changes?**:
No

**If the changes in this PR are manually verified, list down the scenarios covered:**:
The instructions in this doc have been verified manually.

**Any additional information for your reviewer?** : 
This PR should be merged as a release activity after #125 is merged.

### CODE REVIEW

File: docs/tutorial/file-permissions.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 10Gi
+```
+>**Note:** The FilePermissions config may be used with the `cas.openebs.io/config` StorageClass annotation key as well. If the config option is present on both the StorageClass and the PersistentVolumeClaim, the PersistentVolumeClaim config takes precedence.
+
+## FSGID/fsGroup
+
+The 'FSGID' config key (StorageClass annotations) is being deprecated and will be removed in future releases. At present, the FSGID config can be used, but it cannot be used along with FilePermission data keys 'GID' and/or 'mode'.
```
Comment (mittachaitu) [inline docs/tutorial/file-permissions.md]: Good to make bold depreciation note as bold
```suggestion
The 'FSGID' config key (StorageClass annotations) is **being deprecated and will be removed in future releases**. At present, the FSGID config can be used, but it cannot be used along with FilePermission data keys 'GID' and/or 'mode'.
```

**Review by mittachaitu — COMMENTED** [review]
Changes looks good but we also need to update the [troubleshooting doc](https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/docs/troubleshooting/non-root-application-accesing-nfs-volume.md) with related changes

**Comment by mittachaitu** [comment]
Can you also create another PR with helm changes for release?

File: docs/tutorial/file-permissions.md (hunk 1 above)
Comment (niladrih) [reply docs/tutorial/file-permissions.md]: Done :+1:

**Review by mittachaitu — APPROVED** [review]
LGTM

**Pull request merged**